website/app/globals.css

```
@@ -1,116 +1,85 @@
+  --s5: 24px;
+  --s6: 32px;
+  --s7: 48px;
+  --s8: 64px;
+  --s9: 96px;
+
+  --page: 1080px;
+  --gutter: 24px;
+  --nav-h: 56px;
+
+  --sans:
+    var(--font-sans), -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)